test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_CustomFactor.js

```diff
@@ -0,0 +1,46 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2018-06-21T09:27:15.000Z",
+    "status": "MFA_ENROLL_ACTIVATE",
+    "_embedded": {  
+     "user": {  
+       "id": "00upl7WCKsk0Zorcn0g3",
+       "passwordChanged": "2018-06-21T07:15:21.000Z",
+       "profile": {  
+         "login": "xxx@xxx.xxx",
+         "firstName": "xxx",
+         "lastName": "xxx",
+         "locale": "en",
+         "timeZone": "America/Los_Angeles"
+       }
+     },
+     "factor":{  
+       "id":"factorId1234",
+       "factorType":"assertion:saml2",
+       "provider":"GENERIC_SAML",
+       "vendorName":"GENERIC_SAML",
+       "profile":{  
+         "user":"inca@clouditude.net"
+       }
+     }
+    },
+    "_links":{  
+     "next":{  
+       "name":"activate",
```

Comment:
why is this not `redirect`?

Reply:
I'm not sure.. This is how it was designed - https://oktawiki.atlassian.net/wiki/spaces/~ajay.thomas/pages/494863610/Changes+to+authn+response+for+generic+SAML+OIDC+factors#ChangestoauthnresponseforgenericSAML/OIDCfactors-UserenrollsforSAMLfactor